sklearn/pipeline.py

```diff
@@ -876,12 +927,14 @@ def make_union(*transformers, **kwargs):
                               TruncatedSVD(algorithm='randomized',
                               n_components=2, n_iter=5,
                               random_state=None, tol=0.0))],
-           transformer_weights=None)
+           transformer_weights=None, verbose=False)
     """
     n_jobs = kwargs.pop('n_jobs', None)
+    verbose = kwargs.pop('verbose', False)
     if kwargs:
         # We do not currently support `transformer_weights` as we may want to
         # change its type spec in make_union
-        raise TypeError('Unknown keyword arguments: "{}"'
-                        .format(list(kwargs.keys())[0]))
-    return FeatureUnion(_name_estimators(transformers), n_jobs=n_jobs)
+        raise TypeError('Unknown keyword arguments: "{}"'.format(
+            list(kwargs.keys())[0]))
```

Comment:
Please revert the change of the above 2 lines -- this is not an improvement in formatting.